### PR TITLE
added the Strings and localisation info under technical documents

### PR DESCRIPTION
**Pull request by viorel15**
Why?
Lokalise is the new service used for managing texts and translations used in our apps.

### CODE REVIEW

File: TechnicalDocuments/Lokalise.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Lokalise Service
+
+Lokalise is used for managing texts and translations used in our apps.
+
+
+Strings are edited through the lokalise app [https://lokalise.co/](https://lokalise.co/).
```
Comment (diegopetrucci) [inline TechnicalDocuments/Lokalise.md]:
```suggestion
Strings are edited through the [Lokalise app](https://lokalise.co). 
```

File: TechnicalDocuments/Lokalise.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+## Lokalise Service
+
+Lokalise is used for managing texts and translations used in our apps.
+
+
+Strings are edited through the lokalise app [https://lokalise.co/](https://lokalise.co/). 
+Login using your github account and request acces to releavant projects.
```
Comment (diegopetrucci) [inline TechnicalDocuments/Lokalise.md]:
```suggestion
Login using your github account and request access to the relevant projects.
```

File: TechnicalDocuments/Lokalise.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+## Lokalise Service
+
+Lokalise is used for managing texts and translations used in our apps.
```
Comment (diegopetrucci) [inline TechnicalDocuments/Lokalise.md]:
```suggestion
Lokalise is used for managing texts and translations in our apps.
```

File: TechnicalDocuments/Lokalise.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+
```
Comment (diegopetrucci) [inline TechnicalDocuments/Lokalise.md]:
```suggestion
```

File: TechnicalDocuments/Lokalise.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+bundle exec fastlane lokalise_pull         // download all projects
+bundle exec fastlane lokalise_pull_babylon     //download the main Babylon
+
+// these download the other apps projects - they are subsets that populates the files `TargetSpecificLocalizable`
+bundle exec fastlane lokalise_pull_telus
+bundle exec fastlane lokalise_pull_nhs111
+bundle exec fastlane lokalise_pull_bupa```
+```
+
+To add/modify keys/values, when you are in a Project, you can use `⌘K` or click the blue `Add key`…
```
Comment (diegopetrucci) [inline TechnicalDocuments/Lokalise.md]:
```suggestion
To add/modify keys/values, when you are in a `Project` on the website, you can use `⌘K` or click the blue button `Add key`…
```

File: TechnicalDocuments/Lokalise.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+## Lokalise Service
+
+Lokalise is used for managing texts and translations in our apps.
+
+Strings are edited through the [Lokalise app](https://lokalise.co). 
+Login using your github account and request access to the relevant projects.
```
[the remark this answers is not in the document]
Comment (Ben-Henshall) [reply TechnicalDocuments/Lokalise.md]:
```suggestion
Create an account using your `@babylonhealth.com` email address and request access to the relevant projects.
```

**Comment by diegopetrucci** [comment]
@Ben-Henshall I am merging these changes, but your issue is valid. We should reach an agreement on it.

**Pull request merged**